docker-compose.yml

```diff
@@ -25,6 +29,37 @@ services:
       - RESULTSQUEUE=${RABBITMQ_RESULTSQUEUE}
       - WORKERQUEUE=dispatcher
 
+  worker-geoserver-publish-geotiff:
```

Comment:
not that important, but until now we always used the same name for the worker image, the docker container and the `WORKERQUEUE`. I would like to keep it in sync that way, so please rename the container

Reply:
I agree, I changed it